impl/src/main/java/org/apache/myfaces/view/facelets/compiler/SAXCompiler.java

```diff
@@ -631,14 +631,15 @@ public void endDTD() throws SAXException
         @Override
         public void endElement(String uri, String localName, String qName) throws SAXException
         {
+            final boolean nameSpace = CompositeLibrary.NAMESPACE.equals(uri)
```

Comment:
Maybe call the variable `hasNameSpace` or `isInNameSpace` something more explicit that its a boolean?

Reply:
changed.